src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/entity/OffenderEntity.kt

```diff
@@ -55,6 +55,10 @@ class OffenderEntity(
       return person.otherIdentifiers?.crn?.let {
         val offenderEntity = OffenderEntity(
           crn = it,
+          pncNumber = person.otherIdentifiers.pncIdentifier?.pncId,
```

Comment:
👍 although I think we ought to consider at some point persisting the PNCIdentifier type itself here